docs/api_reference/webhooks/introduction.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+Callbell API provides **HTTP Webhooks** for the most common application events:
+
+- **Message events**
+- **Message notification status events**
+- **Contact events**
+
+## How to subscribe to events
+
+On your Callbell dashboard navigate to the [**"API Settings"**](https://dash.callbell.eu/settings/api_settings/webhooks) section and click on the "Webhooks" tab:
+
+![webhook_tab](./assets/webhook_tab.jpg)
+
+In the URL field add your endpoint URL (e.g. `https://myapp.com/callbell_endpoint`) and select the events you want to get subscriptions for:
```

Comment:
I'll also add info regarding the fact the `localhost` in not supported, and that is has to be complete and include `http` or `https`. wdyt?

Reply:
Good point 👍